lib/getModelUploader.js

```diff
@@ -7,6 +7,10 @@ function getModelUploader(port) {
   const { modelName, bootloaderVersion } = port.device;
 
   if (/3100|3111|3121/.test(modelName)) {
+    if (bootloaderVersion === '1.4.2C00') {
+      return uc3100v2;
+    }
+
     if (compareVersions(bootloaderVersion, '1.3.0S07') >= 0) {
```

Comment:
Shouldn't this version be removed? Should we also remove the test files for this version, since it was never released to anything other than a couple boxes we manually installed it on. The uc3100v2 isn't going to work for it now anyways.

Reply:
you are correct, this is a test version and only.